Packages/io.chainsafe.web3-unity.web3auth/Runtime/WalletGUI/Scripts/ShowWalletGui.cs

```diff
@@ -0,0 +1,48 @@
+using System.Threading.Tasks;
+using ChainSafe.Gaming.UnityPackage.Connection;
+using ChainSafe.Gaming.Web3;
+using TMPro;
+using UnityEngine;
+
+namespace ChainSafe.GamingSdk.Web3Auth
+{
+    public class ShowWalletGui : MonoBehaviour, IWeb3InitializedHandler
+    {
+        [SerializeField] private Web3AuthWalletGUI web3AuthWalletGUIPrefab;
+        [SerializeField] private bool displayWalletIcon;
```

Comment:
small suggestion, change all of these (except the prefab itself) to be an SO config.

Reply:
Good idea! we can also move other provider configs to connection provider SOs!

Reply:
but could we maybe do it in another PR since this one is already too big? I'll create an issue for it

Reply:
Yeah sure!

Reply:
https://github.com/ChainSafe/web3.unity/issues/1085 here it is